README.md

```diff
@@ -1 +1,69 @@
 # json-schema-xsd-tools
+<!-- [![npm](https://img.shields.io/npm/v/json-schema-xsd-tools)](https://www.npmjs.com/package/json-schema-xsd-tools) -->
+
+Tools capable of using JSON Schema to generate and validate XSD.
+
+
+JSON Schema is a vocabulary that allows to annotate and validate JSON documents.
+Read more about [JSON Schema](https://json-schema.org/).
+
+
+## Installation
+In a browser:
+
+`<script src="dist/json-schema-xsd-tools.js"></script>`
+
+Using yarn:
+
+`yarn add json-schema-xsd-tools`
```

Comment:
I assume to be able to do this, we need to publish it to npm first, right ? Happy to help with that (or get the resources on the team to do that at least), but until then -  can the readme include a way how to build and include a version in a project locally ? 

In case this is it and yarn/npm can do some magic with locally built packages please let me know (and include such info in this readme).